### PR TITLE
chore(github): Add missing labels to `.github/labels.yml`

### PR DESCRIPTION
**Pull request by ExE-Boss**
This adds the <kbd>[infra 🏗](https://github.com/mdn/browser-compat-data/labels/infra%20%3Abuilding_construction%3A)</kbd> and <kbd>[schema ⚙️](https://github.com/mdn/browser-compat-data/labels/schema%20%3Agear%3A)</kbd> labels to the `.github/labels.yml` file.

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -31,3 +31,8 @@ docs ✍️:
   - "*.md"
 linter 🏡:
   - "test/**"
+infra :building_construction::
```
Comment (ddbeck) [inline .github/labels.yml]: How about sorting this alphabetically?

File: .github/labels.yml [hunk 2]
```diff
@@ -31,3 +31,8 @@ docs ✍️:
...
+  - "/*"
```
Comment (ddbeck) [inline .github/labels.yml]: This is going to catch docs changes (e.g., changes to `README.md` or `GOVERNANCE.md`) and some stuff that I'm not sure we even have a label for (e.g., `index.js`). Enumerating the specific files here might be a better way to go

Comment (ExE-Boss) [reply .github/labels.yml]: Actually, `index.js` and (`types.d.ts`) use the <kbd>[infra :building_construction:](https://github.com/mdn/browser-compat-data/labels/infra%20%3Abuilding_construction%3A)</kbd> label: https://github.com/mdn/browser-compat-data/pull/2338 https://github.com/mdn/browser-compat-data/pull/2308 https://github.com/mdn/browser-compat-data/pull/3739

Same with `package.json`: https://github.com/mdn/browser-compat-data/pull/3681

Comment (ddbeck) [reply .github/labels.yml]: Ah, nice finds! Thank you for clearing that up.

**Pull request merged**